pkg/apis/nodeup/config.go

```diff
@@ -90,6 +90,8 @@ type BootConfig struct {
 	ConfigBase *string `json:",omitempty"`
 	// ConfigServer holds the configuration for the configuration server.
 	ConfigServer *ConfigServerOptions `json:",omitempty"`
+	// APIServer is the API server IP address.
+	APIServer string `json:",omitempty"`
```

Comment:
I know this one is just carrying on the convention, but I do have a hard time understanding when/how this is being used just from looking at the field.

Reply:
True, the comment should be more detailed.